aicssegmentation/tests/test_WorkflowEngine.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from unittest.mock import MagicMock, create_autospec
+import numpy as np
+from aicssegmentation.structure_wrapper.WorkflowEngine import WorkflowEngine
+
+class TestWorkflowEngine:
+    def setup_method(self):
+        self.fake_image = np.zeros([100,100])
+        self.engine = WorkflowEngine("sec61b", self.fake_image)
+
+    def test_get_next_step(self):
+        assert self.engine.get_next_step().name == "Intensity Normalization"
+        self.engine.execute_next()
+        assert self.engine.get_next_step().name == "Edge Preserving Smoothing"
+        self.engine = WorkflowEngine("sec61b", self.fake_image)
```

Comment:
Why is this line necessary? 🤔

Reply:
I'm "resetting" the workflow engine to a new instance so I can test from the start again.

Reply:
Ok, this is totally nit picking now I know... but from what I'm reading it looks like `setup_method` should run at the beginning of every method in this class, so it shouldn't be necessary.

Reply:
Correct, no need to bother resetting anything that's defined by the `setup_method` at the end of a test.

Reply:
Ah I did not know that! I'll make that fix